website/src/components/rsv/RsvSequencingEffortsPage.astro

```diff
@@ -37,7 +37,7 @@ const { label: subdivisionLabel, field: subdivisionField } = getLocationSubdivis
             <div class='max-w-[1000px]'>
                 <LocationTimeFilter
                     fields={view.locationFields}
-                    initialLocation={baselineFilter}
+                    initialLocation={routeData.baselineFilter.location}
```

Comment:
This is confusing:

I was wondering why we don't simply use `baselineFilter.location` instead of `routeData.baselineFilter.location`, but those are completely different types and different concepts.
* The `baselineFilter` is the object that is prepared to be sent as a LAPIS filter
* `routeData.baselineFilter` is the internal state of the website that can be transformed into the `baselineFilter`.

I think we should separate those in terminology to make the distinction clearer between "LAPIS filter" and "dashboard state".

Definitely not something to do in this PR, but to keep in mind.

Reply:
But there is also an issue (from the same confusion) with this PR:

I think we only need the `additionalFilters` in the `View.toLapisFilter` methods, as those represent the transition from "dashboard state" to "LAPIS filter". We don't need the `additionalFilters` in the "dashboard state", since it's already in the config, i.e. we need it neither in `View.parseUrl` functions nor in the `View.defaultRoute`s.

Reply:
True. I changed it, so the additionalFilters are only applied during conversion to a lapisFilter. 
I also agree, that the naming is confusing. I added a ticket for renaming #222